File: easy_34_student_class_display_details/solution.cpp
using namespace std;
#include <iostream>
#include <string>

class Solution {
public:
    string getStudentDetails(const string& name, int roll, float marks) {
        return "Name: " + name + ", Roll: " + to_string(roll) + ", Marks: " + to_string(marks);
    }
};

int main() {
    string name;
    int roll;
    float marks;
    cin >> name >> roll >> marks;
    Solution solution;
    string result = solution.getStudentDetails(name, roll, marks);
    cout << result << endl;
    return 0;
}

File: easy_34_student_class_display_details/solution.py
from typing import TextIO
import sys

class Solution:
    def getStudentDetails(self, name: str, roll: int, marks: float) -> str:
        """
        Returns a formatted string with student details.
        """
        return f"Name: {name}, Roll: {roll}, Marks: {marks}"

def solve(infile: TextIO = sys.stdin, outfile: TextIO = sys.stdout) -> None:
    parts = infile.read().strip().split()
    name = parts[0]
    roll = int(parts[1])
    marks = float(parts[2])
    solution = Solution()
    result = solution.getStudentDetails(name, roll, marks)
    outfile.write(f"{result}\n")

if __name__ == "__main__":
    solve()
